api_spec.yaml

```diff
@@ -1,7 +1,11 @@
 openapi: 3.0.0
 info:
```

Comment:
Add tenantIds to user object itself in APIs that return the user object based on userId, email id etc..

Reply:
Done